packages/exo/test/heap-classes.test.js

```diff
@@ -81,12 +82,23 @@ test('test defineExoClass', t => {
   t.deepEqual(upCounter[GET_INTERFACE_GUARD]?.(), UpCounterI);
   t.deepEqual(getInterfaceMethodKeys(UpCounterI), ['incr']);
 
-  const symbolic = Symbol.for('symbolic');
+  const symbolic = Symbol('symbolic');
   const FooI = M.interface('Foo', {
     m: M.call().returns(),
     [symbolic]: M.call(M.boolean()).returns(),
   });
-  t.deepEqual(getInterfaceMethodKeys(FooI), ['m', Symbol.for('symbolic')]);
+  // Cannot use `t.deepEqual` because it does not recognize that two
+  // unregistered symbols with the same `description` are the same in
+  // our distributed object semantics. Unfortunately, `compareRank` is
+  // too imprecise. We'd like to also test `keyEQ`, but that would violate
+  // our package layering.
```

Comment:
For all these, see if we could use @gibson042 's ordering test instead,

Reply:
I was confused. The code I had in mind is `orderInvariants`. But now that I looked at it, it is not relevant. Resolving.